scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -43,13 +43,11 @@ package object dialects {
 
   @deprecated("Scalameta macro annotations are no longer supported", "4.3.11")
   implicit val Paradise211 = Scala211
-    .withAllowInlineIdents(true)
```

Comment:
Why is this removed?

Reply:
It was added for dotty, before dotty inline could be ident, in dotty it was a keyword used as modifier.
But hey, we have soft keywords lets change inline to be soft keyword in dotty and you can still use it as ident.
Therefere allowInline is always true no exceptions. Backward compatibility though, field will stay with us forever and method will do nothing.